docs/src/modules/components/MarkdownElement.js

```diff
@@ -116,14 +116,17 @@ const styles = theme => ({
       marginTop: -96, // Offset for the anchor.
       position: 'absolute',
     },
-    '& pre, & pre[class*="language-"]': {
```

Comment:
The language class is currently applied to the code element not pre element. The second half of the selector never matches as far as I can tell

Reply:
👍 its no longer needed since I have disabled prism auto highlighting.